scripts/lib/install

```diff
@@ -413,6 +434,14 @@ EOF
         crudini --del /etc/zulip/zulip.conf postgresql
     fi
 
+    if [ -n "$POSTGRESQL_DATABASE_USER" ]; then
+        crudini --set /etc/zulip/zulip.conf postgresql database_user $POSTGRESQL_DATABASE_USER
+    fi
+
+    if [ -n "$POSTGRESQL_DATABASE_NAME" ]; then
+        crudini --set /etc/zulip/zulip.conf postgresql database_name $POSTGRESQL_DATABASE_NAME
+    fi
```

Comment:
Nit: For consistency with other places in this file, swap this around to name, then user.